types/deploy/options/deploy_options.go

```diff
@@ -0,0 +1,19 @@
+package options
```

Comment:
why not move this file to types/provision/deploy_options.go ?

Reply:
I forgot to open the PR as draft, still making changes